.circleci/config.yml

```diff
@@ -426,6 +463,50 @@ jobs:
                   docker tag oqs-openssh-img $TARGETNAME/openssh:latest &&
                   docker push $TARGETNAME/openssh:latest
 
+  ubuntu_x64_envoy:
+    description: Building and pushing OQS-enabled envoy
+    docker:
+      - image: openquantumsafe/envoy:latest
+        auth:
+          username: $DOCKER_LOGIN
+          password: $DOCKER_PASSWORD
+    steps:
+      - checkout # change this from "checkout" to "*localCheckout" when running CircleCI locally
+      - setup_remote_docker
+      - run:
+          name: Authenticate to Docker
+          command: echo $DOCKER_PASSWORD | docker login --username $DOCKER_LOGIN --password-stdin
+      - run:
+          name: Build OQS envoy
+          command: |
+            docker build -t envoy-oqs . 
+          working_directory: envoy
+      - run:
+          name: Build curl with liboqs
+          command: |
+            docker build -t oqs-curl . 
+          working_directory: curl
+      - run:
+          name: Test OQS envoy TLS servers using built OQS envoy
+          command: |
+            ./certs/gen_cert.sh &&
+            docker-compose pull &&
```

Comment:
`docker-compose` is most likely not available in CircleCI: See documentation [here](https://circleci.com/docs/docker-compose).

Reply:
noted and fixed. docker-compose install instructions added to config.yml